app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::BooksController < ApplicationController
+  before_action :load_book, only: :show
+
+  def index
+    @books = Book.includes(:author).order("name ASC")
+    @books = ActiveModelSerializers::SerializableResource.new(@books, {})
+    render json: {status: true, data: {book: @books}}, status: :ok
+  end
+
+  def show
+    render json: {status: true, data: {book: @book}}, status: :ok
+  end
+
+  private
+
+  def load_book
+    @book = Book.includes(:author).find_by id: params[:id]
+    @book = ActiveModelSerializers::SerializableResource.new(@book, {})
+    return if @book
```

Comment:
```suggestion
    @book = Book.includes(:author).find_by id: params[:id]
```

đưa code `ActiveModelSerializers::SerializableResource.new(@book, {})` lên chỗ method show em nhé